plugins/Installation/FormSuperUser.php

```diff
@@ -84,7 +86,7 @@ function init()
  * Login id validation rule
  *
  */
-class Rule_isValidLoginString extends HTML_QuickForm2_Rule
+class RuleIsValidLoginString extends HTML_QuickForm2_Rule
```

Comment:
Separate file? I know it might be tedious for small classes like this, but it's cleaner.